### DISCUSSION TITLE
Is paperless-ng dead?

### DISCUSSION THREAD

**Discussion by stepanov1975** [jonaswinkler/paperless-ng#1689] [General]
Hi.
So from the luck of activity, it seems there will be no further development of paperless-ng?
Am I right?

**Comment by spooner80** [comment]
Hi,

take a look to the new fork
[Paperless-ngx](https://github.com/paperless-ngx/paperless-ngx)

**Comment by stepanov1975** [comment]
@spooner80 Nice!

**Comment by ghost** [comment]
Has Paperless-NG actually been abandoned @jonaswinkler ?
